docs/01-overview/02-main-areas/service-mesh/README.md

```diff
@@ -9,6 +9,7 @@ The main principle of Kyma Service Mesh is to inject Pods of every service with
 By default, Istio in Kyma has [mutual TLS (mTLS)](https://istio.io/docs/concepts/security/#mutual-tls-authentication) enabled and injects a sidecar container to every Pod. You can manage mTLS traffic in services or at a Namespace level by creating [Destination Rules](https://istio.io/docs/reference/config/networking/destination-rule/) and [Peer Authentications](https://istio.io/docs/tasks/security/authentication/authn-policy/). If you disable sidecar injection in a service or in a Namespace, you must manage their traffic configuration by creating appropriate Destination Rules and Peer Authentications.
```

Comment:
Let's discuss with the rest of the TWs during today's call whether we should treat the previous overview docs as new READMEs. As far as I know, Maja has a different approach in [Application Connectivity](https://deploy-preview-11841--kyma-project-docs-preview.netlify.app/docs/kyma/preview/01-overview/02-main-areas/application-connectivity/). Let's discuss.

Reply:
I moved the general description of the Service Mesh are to the README.md. The details were placed in `02-main-areas/service-mesh/smsh-01-details.md`